tests/integration/replication-4.tcl

```diff
@@ -54,30 +54,48 @@ start_server {tags {"repl external:skip"}} {
         test {With min-slaves-to-write (1,3): master should be writable} {
             $master config set min-slaves-max-lag 3
             $master config set min-slaves-to-write 1
-            $master set foo bar
-        } {OK}
+            assert_equal OK [$master set foo 123]
+            assert_equal OK [$master eval "return redis.call('set','foo',12345)" 0]
+            wait_for_condition 50 100 {
+                [string match {*min_slaves_good_slaves:1*} [$master info replication]]
+            } else {
+                fail "Slave didn't sync in time"
+            }
```

Comment:
you changed the wait condition and moved it to after the write, but i still don't understand why it's needed.

Reply:
I think it needed to check the write reached replicas. If not, need we to refactor tests in `replication-2.tcl` also? it has this check after the write.

Reply:
but that test concluded.. so is it needed for the next test to succeed?
the next test waits for 2 replicas, so i think this is irrelevant for it.
did you have any issue for which you added it?

Reply:
Okie, I will remove it.